test/blackbox-tests/test-cases/rule/synopses/multiple-rules-multiple-aliases.t

```diff
@@ -0,0 +1,53 @@
+Dune should show all synopses with their location for all rules to aliases attached.
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.19)
+  > EOF
+  $ cat > dune << EOF
+  > (rule
+  >  (targets file1.ml)
+  >  (aliases rule-with-synopses and-another-rule)
+  >  (synopsis "Rule creates file1.ml")
+  >  (action
+  >    (write-file file1.ml "")))
```

Comment:
You can probably use `%{target}` here, to avoid duplicating `file1.ml`.

Reply:
Done for all tests in [3e66abd](https://github.com/ocaml/dune/pull/11609/commits/3e66abd036e8cba0e21b4d0a269231884ea7b382), thanks!